nltk/util.py

```diff
@@ -277,15 +276,13 @@ def edges2dot(edges, boxes=[], o='default'):
 
     """
     dot_string = 'digraph G {\n'
-    for (source,target) in edges:
-        if o == 'reverse':
-            pair = (target,source)
-        else:
-            pair = (source,target)
-        for box in boxes:
+    for tup in attr.items():
+        dot_string += "%s=%s;\n" % tup
```

Comment:
The [`str.format()`](https://docs.python.org/3/library/string.html#format-examples) methodology for formatting is preferred over the Python 2 `%`-style.
In fact, now that we've deprecated Python 3.5, we can also start using the wonderful [`f-string formatting`](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings).

See the [CONTRIBUTING guidelines](https://github.com/nltk/nltk/blob/develop/CONTRIBUTING.md#code-guidelines).

This also applies a few times below.

Reply:
Thanks @tomaarsen. I have applied all your suggestions.